src/chi/components/popover/popover.scss

```
@@ -203,4 +203,47 @@ $popover-arrow-size: 0.75rem;
       }
     }
   }
+
+  &.-closable {
+    position: relative;
+
+    .chi-popover__content {
+      padding-top: 2.5rem;
+    }
```

This should work and prevent us from having to write a separate rule below for content containers that aren't adjacent siblings of headers. Also, I initially suggested 2.5rem but this should be 2rem - did not see there was a 0.5 margin-top on the child text.

```suggestion
    :not(.chi-popover__header) {
      & + .chi-popover__content {
        padding-top: 2rem;
      }
    }
```